docs/content/en/docs/getting-started/developer/_index.md

```diff
@@ -0,0 +1,281 @@
+
+1. Connect to Cassandra replacing `<k8ssandra-username>` and `<k8ssandra-password>` with the values you retrieved in [Retrieve K8ssandra superuser credentials]({{< relref "/docs/getting-started#superuser" >}}):
+
+    ```bash
+    cqlsh -u <k8ssandra-username> -p <k8ssandra-password>
+    ```
+
+    **Output**:
+
+    ```bash
+    Connected to k8ssandra at 127.0.0.1:9042.
+    [cqlsh 6.8.0 | Cassandra 3.11.6 | CQL spec 3.4.4 | Native protocol v4]
+    Use HELP for help.
+    k8ssandra-superuser@cqlsh>
+   ```
+
+1. Create a new keyspace, `k8ssandra_test`, using [CREATE KEYSPACE](https://docs.datastax.com/en/cql-oss/3.x/cql/cql_reference/cqlCreateKeyspace.html):
+
+    ```sql
+    CREATE KEYSPACE k8ssandra_test  WITH replication = {'class': 'SimpleStrategy', 'replication_factor': 1};
+    ```
+
+1. Switch to the new keyspace using [USE](https://docs.datastax.com/en/cql-oss/3.x/cql/cql_reference/cqlUse.html):
+
+    ```sql
+    USE k8ssandra_test;
+    ```
+
+1. Create a new table, `users` using [CREATE TABLE](https://docs.datastax.com/en/cql-oss/3.x/cql/cql_reference/cqlCreateTable.html#cqlCreateTable)
+
+    ```sql
+    CREATE TABLE users (email text primary key, name text, state text);
+    ```
+
+1. Insert some sample data into the new table using [INSERT](https://docs.datastax.com/en/cql-oss/3.x/cql/cql_reference/cqlInsert.html)
+
+    ```sql
+    INSERT INTO users (email, name, state) values ('alice@example.com', 'Alice Smith', 'TX');
+    INSERT INTO users (email, name, state) values ('bob@example.com', 'Bob Jones', 'VA');
+    INSERT INTO users (email, name, state) values ('carol@example.com', 'Carol Jackson', 'CA');
+    INSERT INTO users (email, name, state) values ('david@example.com', 'David Yang', 'NV');
+    ```
+
+1. Query the data using [SELECT](https://docs.datastax.com/en/cql-oss/3.x/cql/cql_reference/cqlSelect.html) and validate the return results:
+
+    ```sql
+    cqlsh> SELECT * FROM k8ssandra_test.users;
+    ```
+
+    **Output**:
+
+    ```sql
+     email             | name          | state
+    -------------------+---------------+-------
+     alice@example.com |   Alice Smith |    TX
+       bob@example.com |     Bob Jones |    VA
+     david@example.com |    David Yang |    NV
+     carol@example.com | Carol Jackson |    CA
+
+    (4 rows)
+    ```
+
+1. When you're done, exit CQLSH using `QUIT`:
+
+    ```sql
+    cqlsh> QUIT;
+    ```
+
+For complete details on Cassandra, CQL and CQLSH, see the [Apache Cassandra](https://cassandra.apache.org/) web site.
+
+## Next
+
+* For detailed information on additional K8ssandra tasks, see [Tasks]({{< relref "docs/topics" >}}).
+* For a list of frequently asked questions, see the [FAQs]({{< relref "docs/faqs" >}}).
+* For detailed information on K8ssandra, see [Architecture]({{< relref "docs/architecture" >}}).
+* For information on the various K8ssandra Helm charts, see [Helm chart references]({{< relref "docs/reference" >}}).
```

Comment:
There's a lot of `For` in here, can we try and mix it up?

```suggestion
With a running installation check out some of our other guides and docs.

* [Task Guides]({{< relref "docs/topics" >}}).
* [FAQs]({{< relref "docs/faqs" >}}).
* [K8ssandra Architecture]({{< relref "docs/architecture" >}}).
* [Helm chart references]({{< relref "docs/reference" >}}).
```

Reply:
You betcha... a foolish consistency is the hobgoblin of petty minds.